packages/react-core/src/components/Menu/MenuListItem.tsx

```diff
@@ -0,0 +1,179 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Menu/menu';
+import { css } from '@patternfly/react-styles';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/js/icons/external-link-alt-icon';
+import StarIcon from '@patternfly/react-icons/dist/js/icons/star-icon';
+import AngleRightIcon from '@patternfly/react-icons/dist/js/icons/angle-right-icon';
+import CheckIcon from '@patternfly/react-icons/dist/js/icons/check-icon';
+import { MenuContext } from './MenuContext';
+
+export interface MenuListItemProps extends Omit<React.HTMLProps<HTMLLIElement>, 'action' | 'onClick'> {
+  /** Content rendered inside the menu list item. If React.isValidElement(children) props onClick, className and aria-current will be injected. */
+  children?: React.ReactNode;
+  /** Additional classes added to the menu list item */
+  className?: string;
+  /** Identifies the component in the Menu onSelect or onFavorite callback */
+  itemId?: any;
+  /** Target navigation link */
+  to?: string;
+  /** Flag indicating whether the item is active */
+  isActive?: boolean;
+  /** Callback for item click */
+  onClick?: (event: React.MouseEvent) => void;
+  /** Component used to render the menu item */
+  component?: React.ReactNode;
+  /** Render item as disabled option */
+  isDisabled?: boolean;
+  /** Render item with icon */
+  icon?: React.ReactNode;
+  /** Render item with action */
+  action?: React.ReactNode;
+  /** Description of the menu item */
+  description?: string | React.ReactNode;
```

Comment:
```suggestion
  description?: React.ReactNode;
```
string is part of ReactNode already